### ISSUE TITLE
Server Crash

### ISSUE THREAD

**Issue by NikoBellik35** [TeamTwilight/twilightforest#1675]
### Forge Version

43.1.43

### Twilight Forest Version

4.2.1421

### Client Log

https://gist.github.com/NikoBellik35/1e98769dcc24000eb05cd7bf49d57b7f

### Steps to Reproduce

Translated by Yandex. I don't know English well enough.

I do not know how to reproduce this bug. I was just playing with a friend using hamachi. I was thrown off the server every time we approached any boss other than Lich, Naga and Mushroom Minotaur. When we got to the Phantom Knights, I started throwing out here all the time when a friend was around. I asked my friends who make mods. I quote their answer:
"Twilight Forest is crashing. The problem with generating blocks. The problem is at the code level, you can't fix it yourself.
In short, the developer even messed up and mixed up the dots in the code for creating the shape of some block of an object.
This caused the crash of the server, which messed up the generation of new biomes"

### What You Expected

.

### What Happened Instead

.

### Additional Details

.

### Please Read and Confirm The Following

- [X] I have confirmed this bug can be replicated without the use of Optifine.
- [X] I have confirmed the details provided in this report are concise as possible and does not contained vague information (ie. Versions are properly recorded, answers to questions are clear).
- [x] I have confirmed this issue is unique and has not been reported already.

**Comment by GizmoTheMoonPig** [comment]
oh, I mistyped the issue number, this wasnt meant to be closed. 

Unfortunately the log doesnt give me much info, the only things I see are spam about the network sending empty payloads (which I should look into, this is the first time ive seen this) and inverted bounding boxes in a structure (again, never seen this one pop up). Server disconnects are a bit hard to diagnose since often times the log doesnt provide any info for the disconnect. What error message does your screen show when youre disconnected? That may help me diagnose this further. 

Please dont listen to whatever your friend is saying about this, they have zero idea what theyre talking about

**Comment by NikoBellik35** [comment]
Error message
![Uploading IMG_20221029_222653.jpg…]()

**Comment by NikoBellik35** [comment]
![IMG_20221029_222653](https://user-images.githubusercontent.com/85926044/199791779-e6932919-d432-469e-9e95-2359f765f350.jpg)

**Comment by SeraphMint** [comment]
I have a similar issue, on a three-player multiplayer. Same bosses I can't approach, pretty identical error. It ONLY occurs if the Hamachi world host is in the same dimension, otherwise me and the other guy will keep getting kicked out

**Comment by GizmoTheMoonPig** [comment]
Alright, looks like this is a packet issue. Are you playing on a LAN server? I don't see this issue on normal multiplayer anymore but LAN still seems to have the same issue.

**Comment by SeraphMint** [comment]
Hamachi is basically LAN with extra steps, so I think so

**Comment by NikoBellik35** [comment]
Yes. We play on a LAN server. My friend is a host.

**Comment by JetyWawoo** [comment]
Same issue here as with #1634. Hosting on LAN, when I as the host go near my friend when they're near a boss, they get the same error and can't play at all:

> ![IMG_20221029_222653](https://user-images.githubusercontent.com/85926044/199791779-e6932919-d432-469e-9e95-2359f765f350.jpg)

But when I'm not near them they can fight the boss just fine alone.

**Comment by kralrindo** [comment]
Same issue, we both dont have optifine, this error happens when we just explore around and it kicks me from the game

**Comment by GizmoTheMoonPig** [comment]
Try updating forge to 43.1.53, I'm curious if [this PR](https://github.com/MinecraftForge/MinecraftForge/pull/9157) has fixed the issue

**Comment by Andromander** [comment]
Seems like we're getting confirmation that Forge 43.1.53 fixes this problem. If a similar issue occurs and is discovered on this version, feel free to leave a comment, but it may be out of our power. For now, this issue is resolved.
